libs/monorepo-utils/src/cargo.ts

```diff
@@ -4,6 +4,9 @@ import { getAbsoluteRootPath } from './dependencies';
 
 const exec = promisify(execFile);
 
+// Ballot interpreter tests are already run by the pnpm package test job
+const EXCLUDED_PACKAGE_IDS = ['ballot-interpreter'];
```

Comment:
This avoids spinning up a container to re-test `ballot-interpreter`, which is already tested by the pnpm test job for that package. There are a few ways to avoid duplicate test runs:

1. Remove `test:rust-addon` from the `pnpm test` command in `libs/ballot-interpreter/package.json`
2. Special-case `ballot-interpreter` to run in CI with a different command eg. `pnpm test:all-but-rust`
3. This approach

Each has disadvantages
1. Contributors expect to run `pnpm test` to run all tests in the package. Removing `test:rust-addon` from the command may mean contributors are unaware of the Rust tests until they push to CI
2. We need to special-case pnpm test job generation for one repo
3. Essentially special-casing in the opposite direction. Existence of other `test-crate-*` CI jobs implies there should be one for every Rust package